Address review comments in `Content-based message routing` and `Transforming message formats` tutorials (#8248)
ballerina-platform/ballerina-dev-website: swan-lake/integration-tutorials/content-based-message-routing.md

    }
    ```

> 4. Generate record types corresponding to the payload from the hospital backend service by providing samples of the expected JSON payloads.

Reviewer: Need to fix the rest of the sentence too. :)

Author: Oh My bad.

        boolean confirmed;
        string appointmentDate;
    };
    ```

>     Similarly, generate the `ReservationRequest` record type and update the `hospital_id` field with the `HospitalId` type. Remove a one `Patient` record if it gets generated twice.

Reviewer: ```suggestion
    Similarly, generate a record corresponding to the request payload (e.g., `ReservationRequest`) and update the `hospital_id` field to be of the `HospitalId` type that was defined previously. Remove the duplicate `Patient` record since the same type has already been generated.
```

Author: +1. Also, shall we put "Remove the duplicate `Patient` record if the same type has already been generated" (with `if`) since this bug might be fixed in future.

Reviewer: +1.

    }
    ```

    - Define an `http:Client` variable named `hospitalEP`. Later, the relevant client is assigned to this variable based on the `hospital_id` value.

    - Use a [match statement](https://ballerina.io/learn/by-example/match-statement) to assign the relevant client to the `hospitalEP` variable based on the `hospital_id` value.

    - Make a `POST` request to the relevant backend hospital service to make the reservation. The `category` value is used as a path parameter.

    - Use the `is` check to check whether the response is a `ReservationResponse` record, which would indicate a successful reservation. If the response is a `ReservationResponse` record, return the record from the resource method to send a `201 Created` response with the `ReservationResponse` value as the payload.

>     - If the response is not a `ReservationResponse` record (i.e., `http:ClientError`) which indicates that the request failed, log the failure at `ERROR` level.  Return an `http:NotFound` response if the response is  `http:ClientRequestError`, or an `http:InternalServerError` response if the response is `http:ServerError`.

Reviewer: ```suggestion
    - If the response is not a `ReservationResponse` record (i.e., `http:ClientError`) which indicates that the request failed, log the failure at `ERROR` level.  Return an `http:NotFound` response if the response is `http:ClientRequestError`, or an `http:InternalServerError` response if the response is `http:ServerError`.
```

>     - If the response is not a `ReservationResponse` record (i.e., `http:ClientError`) which indicates that the request failed, log the failure at `ERROR` level.  Return a `http:NotFound` response if the response is an `http:ClientRequestError`, or an `http:InternalServerError` response if the response is an `http:ServerError`.

Reviewer: ```suggestion
    - If the response is not a `ReservationResponse` record (i.e., `http:ClientError`) which indicates that the request failed, log the failure at `ERROR` level.  Return an `http:NotFound` response if the response is an `http:ClientRequestError`, or an `http:InternalServerError` response if the response is an `http:ServerError`.
```
